dash-pipeline/SAI/specs/dash_appliance.yaml

```diff
@@ -0,0 +1,35 @@
+!!python/object:utils.sai_spec.sai_api_group.SaiApiGroup
+name: dash_appliance
+description: DASH appliance
+api_type: overlay
+sai_apis:
+- !!python/object:utils.sai_spec.sai_api.SaiApi
+  name: dash_appliance
+  description: DASH appliance
+  is_object: true
+  enums: []
+  structs: []
+  attributes:
+  - !!python/object:utils.sai_spec.sai_attribute.SaiAttribute
+    name: SAI_DASH_APPLIANCE_ATTR_LOCAL_REGION_ID
```

Comment:
can we add this attributes to SAI switch instead of creating a new object?

the switch attributes are essentially for all the global attributes.

Reply:
Adding it to switch object may result in additional dependencies for switch object. For eg: if local region ID is immutable, then this would mean postponing SAI switch create until this value is available. Hence better to avoid this if possible.

Reply:
I think immutable is not really the reason. The main reason of not putting it in the SAI switch is because this ID is coming from SDN controller, which is not available when SAI switch is created.